workspace/lib/lita/handlers/teamcity.rb

```diff
@@ -150,6 +153,15 @@ class Teamcity < Handler
           t('help.artifacts.syntax_specific') => t('help.artifacts.desc_specific')
         }
       )
+
+      route(
+        /^chewie_artifacts\s#{CHEWIE_PR_ID_PATTERN}$/,
```

Comment:
```get_artifacts <key_name>```
key_name is from a Map that is an ENV variable defined in ansible-bot-deployment repo. For example:
```
{"chewie_build": "http://10.10.7.40:8081/artifactory/chewie-test/chewie/PR/"}
```